### PR TITLE
RELATED: SD-933 Update changelog for 7.5.0

### PR DESCRIPTION
**Pull request by naibinh**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks
- SD-933: https://jira.intgdc.com/browse/SD-933

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7123/ : SUCCESS in 4m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3100/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7124/ : SUCCESS in 4m 25s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3101/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7147/ : SUCCESS in 6m 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3126/ : SUCCESS in 3s

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,17 @@ The REST API versions in the table are just for your information as the values a
 
 - We ask developers to consider using the Headline component instead of the KPI component. The KPI component may be eventually marked as deprecated in one of the next major versions.
 
+## 7.5.0
+
+### Added
+
+- Geo charts (pushpin) component ([doc](https://sdk.gooddata.com/gooddata-ui/docs/geo_pushpin_chart_component.html))
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md:24]: - Geo pushpin chart component ([doc](https://sdk.gooddata.com/gooddata-ui/docs/geo_pushpin_chart_component.html))

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,6 +17,17 @@ The REST API versions in the table are just for your information as the values a
...
+- Measure value filter dropdown with button component in SDK
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md:25]: - Custom toggle button for the measure value filter dropdown

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,6 +17,17 @@ The REST API versions in the table are just for your information as the values a
...
+
+### Fixed
+
+- Visualization rendering after error that persisted even though there was no longer an error
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md:29]: - Visualization rendering stuck due to an error that persisted after the error was gone

**Comment by evgeniabogranskaya** [comment]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3734/ : SUCCESS in 23m 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7152/ : SUCCESS in 20m 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1732/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2549/ : SUCCESS in 22m 36s

**Pull request merged**